rep-0142.rst

```diff
@@ -59,13 +59,14 @@ ROS Core
 
 The `ros_core` metapackage composes the core communication protocols.
 It may not contain any GUI dependencies.
+In ROS Jade ros_core is extended to include geneus.
```

Comment:
Please quote the (meta)package names:

    In ROS Jade `ros_core` is extended to include `geneus`.